acceptance/bundle/unitycatalog/capture_schema_dependency/dlt_schema/databricks.yml

```diff
@@ -0,0 +1,43 @@
+bundle:
```

Comment:
> [cli/bundle/config/mutator/capture_schema_dependency_test.go](https://github.com/databricks/cli/blob/2e70558dc18fe4290f62e2a87db613fb9d4827e5/bundle/config/mutator/capture_schema_dependency_test.go#L1)
> Line 1 in [2e70558](https://github.com/databricks/cli/commit/2e70558dc18fe4290f62e2a87db613fb9d4827e5)
 package mutator
 
 Do we need the unit tests? Is there something they cover that this acceptance tests won't?

Reply:
> Do we need the unit tests? Is there something they cover that these acceptance tests won't?

No, mostly. They're local to the mutator itself, which implements the functionality, and that's why I did not remove them. Unit tests next to the code are nice to have and can live independently of the acceptance tests.

There's some additional coverage for what happens when the resources pointers are `nil`, but nothing that absolutely needs to be covered, IMO.